src/hooks/UrqlUseSubscription.re

```diff
@@ -17,20 +17,21 @@ type useSubscriptionArgs = {
   context: UrqlClient.ClientTypes.partialOperationContext,
 };
 
+type executeSubscriptionJs;
+
 [@bs.module "urql"]
 external useSubscriptionJs:
   (useSubscriptionArgs, option((option('acc), Js.Json.t) => 'acc)) =>
-  array(UrqlTypes.jsResponse('ret, 'extensions)) =
+  (UrqlTypes.jsResponse('ret, 'extensions), executeSubscriptionJs) =
   "useSubscription";
 
 /**
  * A function for converting the response to useQuery from the JavaScript
  * representation to a typed Reason record.
  */
-let useSubscriptionResponseToRecord =
-    (parse, result): UrqlTypes.hookResponse('response, 'extensions) => {
-  let data =
-    result->UrqlTypes.jsDataGet->Js.Nullable.toOption->Belt.Option.map(parse);
+let jsSubscriptionResponseToRecord =
```

Comment:
We do have a consistent name for this function in other modules:

```suggestion
let urqlResponseToReason =
```

Mind updating it just to keep the nomenclature consistent and scannable across these hooks?